### ISSUE TITLE
Changes needed in HTML & CSS

### ISSUE THREAD

**Issue by iamdopecode** [incubateind/gdtc-Day2#115]
@rexyagami some changes are needed because there are some design flaws that I have to change

**Comment by rexyagami** [comment]
@shubham13202 yes you have to make changes in the HTML & CSS file in order to achieve the final design.
